### DISCUSSION TITLE
PLEASE ADD MORE

### DISCUSSION THREAD

**Discussion by rahulc07** [Zodsmar/SeedSearcherStandaloneTool#67] [Ideas]
I don't know if you can do all of these....

1.Search for high eyed end portals eg 12 eye.
2.Look for biomes where you want them.  EG.A mushroom biome at spawn with a mesa 100 blocks away.
3.Why is there no stronghold structure?
4.Thanks

**Comment by Zodsmar** [comment]
1. I would like to add this at some point but it definitely won't be for a while. I have many other things that need to be implemented first.
2. This is very high on the list and is something I want to implement very very soon.
3. Strongholds have a set spawning area and because of this, we know 100% always roughly where they are. Also, stronghold searching is very slow so there isn't really a point to add it. Reference here: https://github.com/Zodsmar/SeedSearcherStandaloneTool/wiki/Stronghold-Searching
4. no problem

**Comment by Zodsmar** [comment]
Technically 2 is now possible by searching a predefined list of seeds. Search for Mushroom at spawn, take all those seeds it gave out and pass it back in with new parameters. #74 Added this
